src/App.tsx

```diff
@@ -3,20 +3,45 @@ import {
   ChakraProvider,
   Box,
   Text,
-  Link,
-  VStack,
-  Code,
-  Grid,
   theme,
+  NumberInput,
+  Modal,
+  ModalOverlay,
+  ModalContent,
+  ModalHeader,
+  ModalFooter,
+  ModalBody,
+  ModalCloseButton,
+  useDisclosure,
+  Button,
+  Input,
+  Stack,
+  NumberInputField,
+  NumberInputStepper,
+  NumberIncrementStepper,
+  NumberDecrementStepper
 } from "@chakra-ui/react"
-import { ColorModeSwitcher } from "./ColorModeSwitcher"
-import { Logo } from "./Logo"
+import {CreateCategoryModal} from "./CreateCategoryModal"
+
+function ModalImplementation() {
+  const [goals, setGoals] = React.useState(["Working out", "Studying", "Reading"]);
+
+  return (
+    <Stack>
+      {goals.map((goal, index) => (
+        <Text key={index}>{goal}</Text>
+      ))}
+      <CreateCategoryModal goals = {goals}/>
+    </Stack>
+  )
+}
```

Comment:
Was this made to just test out the modal? I feel like we should remove this since we're going to be storing `goals` in the global state using `hookState`  (for now anyways)

Reply:
Yep just to test out the modal